src/main/java/com/solvd/automation/lab/carina/demo/api/azure/user/interfaces/AzureUserMethod.java

```diff
@@ -0,0 +1,12 @@
+package com.solvd.automation.lab.carina.demo.api.azure.user.interfaces;
+
+import com.qaprosoft.carina.core.foundation.utils.Configuration;
+
+public interface AzureUserMethod {
```

Comment:
just use constant class for it, and call it like PropertyConstant.AZURE_USERNAME. no need to implement the interface